core-web/apps/dotcms-ui/src/app/view/components/dot-edit-layout-designer/components/dot-edit-layout-grid/dot-edit-layout-grid.component.spec.ts

```diff
@@ -22,6 +22,7 @@ import {
     PaginatorService
 } from '@dotcms/data-access';
 import { NgGridModule } from '@dotcms/dot-layout-grid';
+// eslint-disable-next-line @nx/enforce-module-boundaries
```

Comment:
Why? rules is a lib as well.

Reply:
Is weird that `DotAutofocusModule` was in rules.

Reply:
Fixed, used the wrong `DotAutofocusModule ` instead of standalone `DotAutofocusDirective`